### ISSUE TITLE
Trouble installing pencil blue on ubuntu

### ISSUE THREAD

**Issue by Greatnesssurroundsme** [pencilblue/pencilblue#793]
I have barely any experience using github and I didn't know where else to ask.. I saw you can label an issue with "help wanted" but not sure how to add a label, just hope that you don't think this is too trivial question to ask for help. 

Here's a screenshot of my error log: 
http://i58.tinypic.com/212zac8.jpg

I understand already that the folder that I want to install it to shouldn't exist already from reading the fatal warning but is that what is causing the errors?

I've blanked out some info to keep my privacy because it feels a bit embarrassing to ask for help about installing something :P 

inside the html folder is where I have the index.html file. 
Was I suppsed to cd so that I'm inside the html folder and then "pbctrl install a-new-folder"?
Or should I have removed the html folder and then "pbctrl install html"?

**Comment by brianhyder** [comment]
Hey @Greatnesssurroundsme, we're happy to help.  We get questions from all skill levels and putting your question on GitHub helps us to track the most frequent questions as well as make it easier for others to search for similar questions/answers.  

The installer expects to create its own directory.  I would not recommend installing pencilblue under the ```var/www/html``` as it is the default directory for apache.  PencilBlue does not rely on Apache and therefore you could put yourself at risk for exposing your source files if you install it there.

I would recommend installing in a different location.  PencilBlue is self contained so it can be installed anywhere on the system that you have write access to.

Side note: PencilBlue will not start the database for you.  Once you complete the install you must start ensure your mongo instance is running before starting pencilblue.

**Comment by blakecallens** [comment]
@Greatnesssurroundsme here's a [step by step tutorial for Ubuntu](https://github.com/pencilblue/pencilblue/wiki/Quickstart:-Installation#-linux-installation).

**Comment by Greatnesssurroundsme** [comment]
I installed it to html/pb instead, there were a few errors during installation regarding something called kerberos but everything else installed successfully it seems. 

But I'm not sure how to get to the website? there's no index.html included in the install?
In the tutorial video everything is already installed and he has already got the basic website showing and has admin panel open etc. 

I changed my nginx configuration for this server block to use the root /var/www/domain.com/html/pb but it also wants to know the index file which I currently have index.html and index.htm in. 

Also did sudo chown -R $USER:$USER /var/www/domain.com/html/pb 

Thanks again, I hope I'm not over my head here with pencilblue.. I'm still learning about the mean stack and I thought its more interesting to use pencilblue as my cms instead of a php based cms such as wordpress because of that.
